index.js

```diff
@@ -0,0 +1,5 @@
+function strictEquals(val1, val2) {
+  return val1 == val2 && typeof val1 == typeof val2
+}
```

Comment:
Nice clean implementation. You could also write this in ES6+ syntax and take advantage of fat arrow's implicit return.

```JavaScript
const strictEquals = (val1, val2) => val1 == val2 && typeof val1 == typeof val2
```